srv/components/system/config/service_user.sls

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
Where are we calling this state from?

Reply:
Good catch. I will add the trigger.

Reply:
[done](https://github.com/Seagate/cortx-prvsnr/blob/5eb884a11fc622c405fd44a396242bf6e43e16a8/srv/components/system/config/init.sls#L23)